v0.1/core/visualizationParameters.schema.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$id": "https://schema.hbp.eu/sand/0.1/core/visualizationParameters.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Definition of visualization parameters",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "@type",
+    "@id",
+    "bestViewPoint",
+    "definedIn",
+    "internalFileIdentifier"
+  ],
+  "properties": {
```

Comment:
Preferably, define the type for each property to be consistent (even though `"type": "object"` is redundant).

Reply:
see comment above (despite what we thought before)